third_party/cargo/remote/wasi-0.10.0+wasi-snapshot-preview1.BUILD

```diff
@@ -0,0 +1,44 @@
+"""
+@generated
+cargo-raze crate build file.
+
+DO NOT EDIT! Replaced on runs of cargo-raze
+"""
+
+package(default_visibility = [
+    # Public for visibility by "@raze__crate__version//" targets.
+    #
+    # Prefer access through "//third_party/cargo", which limits external
+    # visibility to explicit Cargo.toml dependencies.
+    "//visibility:public",
+])
+
+licenses([
+    "notice",  # Apache-2.0 from expression "Apache-2.0 OR (Apache-2.0 OR MIT)"
+])
+
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_test",
+)
+
+rust_library(
+    name = "wasi",
```

Comment:
ooc, what do you depend on WASI for?

Reply:
this is actually not an intentional dependency, it's being pulled in because raze compiles all child crates with all features enabled, so in the strictest sense is not needed. That said, I don't see an option in raze to disable that, so it'll likely have to be added for now

Reply:
no worries, i just think WASI is cool and was interested to hear if you had found a use case for it.